File: Modul-1/python/PRAK110-2310817110006-Adrian_Bintang_Saputera.py
import math

alas = int(input("Masukkan Alas: "))
tinggi = int(input("Masukkan Tinggi: "))

sisiA = tinggi
sisiB = int(math.sqrt(alas**2 + tinggi**2))
sisiC = alas

keliling = sisiA + sisiB + sisiC
luas = int(0.5 * alas * tinggi)

print("Diketahui :")
print(f"Alas = {alas} cm")
print(f"Tinggi = {tinggi} cm")

print("\nJawab :")
print(f"Sisi A = {sisiA} cm")
print(f"Sisi B = {sisiB} cm")
print(f"Sisi C = {sisiC} cm")
print(f"Keliling = {keliling} cm")
print(f"Luas = {luas} cm")

File: Modul-1/C/PRAK110-2310817110006-Adrian_Bintang_Saputera.c
#include <stdio.h>
#include <math.h>

int main()
{
    int alas, tinggi, sisiA, sisiB, sisiC, keliling, luas;
    printf("Masukkan Alas: ");
    scanf("%d", &alas);
    printf("Masukkan Tinggi: ");
    scanf("%d", &tinggi);

    sisiA = tinggi;
    sisiB = sqrt(pow(alas, 2) + pow(tinggi, 2));
    sisiC = alas;

    keliling = sisiA + sisiB + sisiC;
    luas = 0.5 * alas * tinggi;

    printf("Diketahui :\n");
    printf("Alas = %d cm\n", alas);
    printf("Tinggi = %d cm\n", tinggi);

    printf("\nJawab :\n");
    printf("Sisi A = %d cm\n", sisiA);
    printf("Sisi B = %d cm\n", sisiB);
    printf("Sisi C = %d cm\n", sisiC);
    printf("Keliling = %d cm\n", keliling);
    printf("Luas = %d cm\n", luas);

    return 0;
}